packages/storybook/src/schematics/configuration/configuration.ts

```diff
@@ -106,5 +107,24 @@ function addStorybookTask(projectName: string, uiFramework: string): Rule {
         }
       }
     });
+    projectConfig.targets.set('storybook-build', {
```

Comment:
nitpick:

Can you please set the target as `build-storybook` so it's a verb like other target names?

Reply:
ofc, name was already changed to `storybook-build` to get compliant